### DISCUSSION TITLE
OSError : handle is closed

### DISCUSSION THREAD

**Discussion by IndrajeetTech2020** [apache/airflow#19300] [Q&A]
When I am trying to run dags ,I am getting 

OSError : handle is closed

and dag run breaks immediately.

I am using airflow version 2.1.0 with postgres database . I am not using gpu systems but my program requires gpu system.

When i am running locally , it works with worning messange that you do not have a gpu setup and works.but in airflow its not working.

**Comment by mik-laj** [comment]
It may be related to: https://github.com/apache/airflow/issues/16140
